src/components/Messages/Messages.js

```diff
@@ -1,13 +1,18 @@
 import React from "react";
+import { fetchChannelMessages } from "../../utils/fetcher";
+
+export const Messages = ({ currentChannel }) => {
+  let messages = fetchChannelMessages(currentChannel);
 
-const Messages = () => {
   return (
     <div>
-      <header>
-        <h1>Messages</h1>
-      </header>
+      <h1>{currentChannel}</h1>
```

Comment:
Why are you defining the `currentChannel` twice?

Reply:
removed the one from Channels View and keeping this one in Messages